.github/workflows/typo3_v11ci.yml

```diff
@@ -0,0 +1,38 @@
+# Adapted from https://github.com/TYPO3GmbH/blog/blob/master/.github/workflows/ci.yml
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
```

Comment:
Remove return. If you want you can seperate properties on same level with a return like after runs-on

Reply:
Done